translations/translations.json

```diff
@@ -1227,32 +1227,20 @@
   },
   "feedback": {
     "de": {
-      "technicalTopics": "Technische Funktionen",
-      "contentOfCity": "Inhalte von {{city}}",
-      "contentOfOffer": "Angebot \"{{offer}}\"",
       "disclaimer": "Kontakt und Impressum",
-      "searchFor": "Suche nach",
-      "contentOfPoi": "Inhalt des Ortes \"{{poi}}\"",
-      "contentOfPage": "Inhalt der Seite \"{{page}}\"",
-      "contentOfEvent": "Inhalt der Veranstaltung \"{{event}}\"",
       "send": "Abschicken",
       "failedSendingFeedback": "Beim Senden des Feedbacks ist ein Fehler aufgetreten.",
       "feedback": "Feedback",
       "positiveComment": "Was war hilfreich?",
       "negativeComment": "Was können wir besser machen?",
-      "feedbackType": "Wofür ist das Feedback?",
       "isThisSiteUseful": "Ist diese Seite hilfreich?",
       "useful": "Hilfreich",
       "notUseful": "Nicht hilfreich",
       "nothingFound": "Es wurden leider keine passenden Ergebnisse gefunden.",
-      "yourFeedback": "Ihr Feedback...",
-      "informationNotFound": "Nicht das Richtige gefunden?",
       "wantedInformation": "Welche Informationen fehlen hier?",
       "feedbackSent": "Feedback abgeschickt",
       "thanksMessage": "Vielen Dank für Ihr Feedback. Damit helfen Sie uns, unsere Inhalte und die {{appName}}-App zu verbessern. Wir würden uns über weiteres Feedback freuen.",
       "close": "Schließen",
-      "negativeRating": "Diese Seite ist nicht hilfreich",
```

Comment:
I think this is still used on web, best check other translations as well before deleting.

Reply:
good point, forgot about checking usages in web. Done now.